lib/collection/src/operations/types.rs

```diff
@@ -1366,16 +1386,22 @@ impl SparseIndexParams {
         SparseIndexParams {
             full_scan_threshold,
             on_disk,
+            datatype: None, // TODO
```

Comment:
Shall we propagate it here from parameters?

Reply:
Fixed. (this method was not used, so I've deleted it at all)